### PR TITLE
don't let tooltips grow forever

### PR DESCRIPTION
**Pull request by alexkrolick**
Thank you for contributing to Reach UI! Please fill in this template before submitting your PR to help us process your request more quickly.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code (Compile and run).
- [ ] Add or edit tests to reflect the change (Run with `yarn test`).
- [ ] Add or edit Storybook examples to reflect the change (Run with `yarn start`).
- [ ] Ensure formatting is consistent with the project's Prettier configuration.

This pull request:

- [ ] Creates a new package
- [ ] Fixes a bug in an existing package
- [x] Adds additional features/functionality to an existing package
- [ ] Updates documentation or example code
- [ ] Other

If creating a new package:

- [ ] Make sure the new package directory contains each of the following, and that their structure/formatting mirrors other related examples in the project:
  - [ ] `examples` directory
  - [ ] `src` directory with an `index.tsx` entry file
  - [ ] At least one example file per feature introduced by the new package
  - [ ] Base styles in a `style.css` file (if needed by the new package)

### CODE REVIEW

File: packages/tooltip/styles.css [hunk 1]
```diff
@@ -13,4 +13,5 @@
   background: #f0f0f0;
   color: #444;
   border: solid 1px #ccc;
+  max-width: 12.5em;
```
Comment (alexkrolick) [inline packages/tooltip/styles.css]: = 200px at browser default of 16px per em

**Comment by chancestrickland** [comment]
Is there a reason this isn't better left for folks to handle in their app? We try to keep styles somewhat minimal.

**Comment by alexkrolick** [comment]
> There are some basic styles to make the components usable off-the-shelf

I think most people don't expect a tooltip to flow all the way off the screen for long text. I certainly didn't, and it's not that easy to debug a transient DOM element in the devtools when it's visibility is controlled by JS and not :hover states.

**Comment by chancestrickland** [comment]
I still think this makes more sense for apps to handle, as there are a number of ways you might want to deal with this and I don't think we should be opinionated about the solution. In general you should probably avoid tooltips with enough content to trigger this problem in the first place, but if you do need this solution you can add a max-width to your stylesheet or handle it with a custom positionig function with the `position` prop on `TooltipContent`.

**Pull request closed without merging**